drivers/dma/dma_intel_adsp_hda.c

```diff
@@ -183,7 +184,7 @@ int intel_adsp_hda_dma_host_reload(const struct device *dev, uint32_t channel,
 
 #if CONFIG_DMA_INTEL_ADSP_HDA_TIMING_L1_EXIT
 #if CONFIG_SOC_SERIES_INTEL_ACE
```

Comment:
You can remove this condition since you are already doing it in the implementation

Reply:
Done